From 53ac54bee550e70cfa58a600c4dd8f41e3488ef9 Mon Sep 17 00:00:00 2001
From: Sameer Puri <purisame@spuri.io>
Date: Mon, 13 Apr 2020 17:51:47 +0200
Subject: [PATCH 620/921] [Backport] media: i2c: imx219: remove redundant
 writes

stable inclusion
from stable-v5.10.173
commit 09a0410886aa80269d5fee39a5d61dde98694f90
category: bugfix
bugzilla: https://gitee.com/openeuler/kernel/issues/I6PMQI

Reference: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=09a0410886aa80269d5fee39a5d61dde98694f90

--------------------------------

[ Upstream commit fbef89886da6d7735d20fdde16a1ee6ed6c6ab56 ]

These writes to 0x162, 0x163 already appear earlier in the struct for
the 1920x1080 mode and do not need to be repeated.

Signed-off-by: Sameer Puri <purisame@spuri.io>
Reviewed-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
Signed-off-by: Mauro Carvalho Chehab <mchehab+huawei@kernel.org>
Stable-dep-of: ef86447e775f ("media: i2c: imx219: Fix binning for RAW8 capture")
Signed-off-by: Sasha Levin <sashal@kernel.org>
Signed-off-by: Wang Hai <wanghai38@huawei.com>
---
 drivers/media/i2c/imx219.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/media/i2c/imx219.c b/drivers/media/i2c/imx219.c
index 4771d0ef2c46..cad0a8df203e 100644
--- a/drivers/media/i2c/imx219.c
+++ b/drivers/media/i2c/imx219.c
@@ -262,8 +262,6 @@ static const struct imx219_reg mode_1920_1080_regs[] = {
 	{0x4793, 0x10},
 	{0x4797, 0x0e},
 	{0x479b, 0x0e},
-	{0x0162, 0x0d},
-	{0x0163, 0x78},
 };
 
 static const struct imx219_reg mode_1640_1232_regs[] = {
-- 
2.17.1

